Firmware/Marlin_main.cpp

```diff
@@ -3828,7 +3853,10 @@ static void gcode_M600(bool automatic, float x_position, float y_position, float
 	fsensor_check_autoload();
 #endif //IR_SENSOR
 
-    lcd_setstatuspgm(MSG_WELCOME);
+    if (isPrintPaused)
+        lcd_setstatuspgm(_i("Print paused"));////MSG_PRINT_PAUSED c=20 r=1
```

Comment:
Doesn't this overwrite the M117 message?

Reply:
I see now my comment doesn't really make sense in the context of this PR.

When a M600 is issued during a paused print, it doesn't really matter if the M117 was called prior.